pynucastro/networks/rate_collection.py

```diff
@@ -923,7 +923,19 @@ def make_ap_pg_approx(self, intermediate_nuclei=None):
 
     def evaluate_rates(self, rho, T, composition, screen_func=None):
         """evaluate the rates for a specific density, temperature, and
-        composition, with optional screening"""
+        composition, with optional screening
+
+        Note: this returns that rate as dY/dt, where Y is the molar fraction.
+        For a 2 body reaction, a + b, this will be of the form:
+
+        rho Y_a Y_b N_A <sigma v> / (1 + delta_{ab}
+
+        where delta is the Kronecker delta that accounts for a = b.
+
+        If you want dn/dt, where n is the number density (so you get
+        n_a n_b <sigma v>), then you need to multiply the results here
+        by rho N_A (where N_A is Avogadro's number).
```

Comment:
I think we usually define it as X/A, where A is just the atomic weight.

But we might want to check if we are being consistent and / or what others do.

Reply:
right, but I guess my concern is when converting Y to n, we get rid of mass in density by N_A/A here not N_A/A_nuc, so I think its off by a ratio between A and A_nuc?

Reply:
realizing this issue can also come up when calculating enuc?

Reply:
it could be.  I think a lot of sources don't worry about the difference between A and A_nuc.  I think we are approaching the point where it would be good to go over the docs and check to see how consistent we are and add some docs describing our assumptions, etc. and then come up with a plan on how to fix this.

Could be a hackathon...